internal/bminventory/inventory.go

```diff
@@ -1589,6 +1598,16 @@ func (b *bareMetalInventory) updateClusterData(ctx context.Context, cluster *com
 	if err = network.VerifyClusterCIDRsNotOverlap(machineCidr, clusterCidr, serviceCidr, userManagedNetworking); err != nil {
 		return common.NewApiError(http.StatusBadRequest, err)
 	}
+
+	if params.ClusterUpdateParams.Operators != nil && params.ClusterUpdateParams.Operators.Lso != nil {
+		updates["operators"] = *params.ClusterUpdateParams.Operators
+		updates["operators_lso"] = *params.ClusterUpdateParams.Operators.Lso
+		updates["operators_lso_enabled"] = *params.ClusterUpdateParams.Operators.Lso.Enabled
```

Comment:
Don't you need just this part in the updates list?

Reply:
@priyanka19-98 please fix accordingly

Reply:
@filanov , any other changes to be made in this part?

Reply:
i don't think that the db schema have thise:
```
		updates["operators"] = *params.ClusterUpdateParams.Operators
		updates["operators_lso"] = *params.ClusterUpdateParams.Operators.Lso
```

Reply:
Made the changes